TeslaLib/TeslaClient.cs

```diff
@@ -141,7 +141,7 @@ public async Task LoginUsingTokenStoreAsync(string password, string mfaCode = nu
 
             TimeSpan expirationTimeFromNow = token.ExpiresUtc - DateTime.UtcNow;
 
-            if (expirationTimeFromNow < TokenExpirationRenewalWindow || forceRefresh)
+            if (expirationTimeFromNow < TokenExpirationRenewalWindow || forceRefreshOlderThanToday)

```

Comment:
In practice I think access tokens are now valid for 8 hours, while refresh tokens re available for longer.  So the "older than today" part doesn't make sense.  Also, the code here will do a refresh if this parameter is set.